[INFO -- 2017-03-07 22:19:38]
Starting flip_value script
version: 2.0
[INFO -- 2017-03-07 22:19:38]
Initialization strings:
[INFO -- 2017-03-07 22:19:38]
(1) target extended-remote | ssh -T mic0 gdbserver --multi -
[INFO -- 2017-03-07 22:19:38]
(2) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-03-07 22:19:38]
(3) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-03-07 22:19:38]
(4) file /micNfs/gdb-fault-injection/injector/bin/lud/lud_check
[INFO -- 2017-03-07 22:19:38]
(5) set remote exec-file /micNfs/gdb-fault-injection/injector/bin/lud/lud_check
[INFO -- 2017-03-07 22:19:38]
(6) set args -s 2048 -n 228 -i /micNfs/gdb-fault-injection/injector/bin/lud/input_2048_th_228 -g /micNfs/gdb-fault-injection/injector/bin/lud/gold_2048_th_228
[INFO -- 2017-03-07 22:19:38]
initSignal:4
[INFO -- 2017-03-07 22:19:38]
endSignal:6
[INFO -- 2017-03-07 22:19:38]
seqSignal:30
[INFO -- 2017-03-07 22:19:38]
faultMode:2
[EXCEPTION -- 2017-03-07 22:19:49]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 22:19:49]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 22:19:49]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 22:19:49]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 22:19:49]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 22:19:49]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 22:19:49]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 22:19:49]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 22:19:49]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 22:19:49]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 22:19:49]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 22:19:49]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 22:19:49]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 22:19:49]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 22:19:49]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 22:19:49]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 22:19:49]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 22:19:49]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 22:19:49]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 22:19:49]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 22:19:49]
pythonException: invalid frame
[EXCEPTION -- 2017-03-07 22:19:49]
pythonException: invalid frame
[INFO -- 2017-03-07 22:19:49]
Backtrace BEGIN:
#0  0x0000000000404cfe in L_lud_omp_123__par_loop2_2_5 () at lud_omp.c:148
#1  0x00007ffff790ba1f in ?? ()
#2  0x00007fffffffd9c0 in ?? ()
#3  0x00007ffff4ea0100 in ?? ()
#4  0x0000000000000001 in ?? ()
#5  0x0000000000000003 in ?? ()
#6  0x0000000000000000 in ?? ()

143	            for (i = 0; i < BS; i++)
144	            {
145	                for (k=0; k < BS; k++) {
146	#pragma omp simd 
147	                    for (j = 0; j < BS; j++) {
148	                        sum[j] += temp_left[BS*i + k] * temp_top[BS*k + j];
149	                    }
150	                }
151	#pragma omp simd 
152	                for (j = 0; j < BS; j++) {

Backtrace END
Memory content before bitflip:0x040x110x000x00
Fault Mode: Random bit-flip
base address to flip value: 0x7fffffffd934
address max offset: 4
Memory content after  bitflip:0x130x200x5a0xe3
frame name: L_lud_omp_123__par_loop2_2_5
symbol name: chunks_per_inter
symbol filename: lud_omp.c
symbol line: 45
value: -480632813
value address: 0x7fffffffd934
Type: The type is an integer type.
Type sizeof: 4

[INFO -- 2017-03-07 22:19:49]
Fault Injection Successful after 11.2235910892s
[INFO -- 2017-03-07 22:19:54]
event type: exit
[INFO -- 2017-03-07 22:19:54]
exit code: 0
